history/titles/c_asova.txt

```diff
@@ -24,5 +24,9 @@
 476.02.15={holder=270667} #Zaebobhen[121004]
 523.01.18={holder=270673} #Jebhaqur[121004]
 534.08.04={holder=270676} #Chabhasoth[121004]
-550.10.09={holder=270680} #Baenrugo[121004]
+# Suntouched takeover
+550.10.09={
+	liege=d_asova
+	holder=273101 #Husam
+}
```

Comment:
But Husam is Titanic.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>